src/inference/dev_api/threading/ie_istreams_executor.hpp

```diff
@@ -88,7 +88,7 @@ class INFERENCE_ENGINE_API_CLASS(IStreamsExecutor) : public ITaskExecutor {
         static void UpdateHybridCustomThreads(Config& config);
 
         std::string _name;          //!< Used by `ITT` to name executor threads
-        int _streams = 1;           //!< Number of streams.
+        int _streams = 0;           //!< Number of streams.
```

Comment:
It was originally changed to 0 because the default input is 0 when calculating streams. At present, it has been changed back to 1 because the ci test failed.